.github/workflows/test-win.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 jobs:
   test:
-    runs-on: windows-2019
+    runs-on: windows-latest
```

Comment:
this must stay windows-2019 to remain compatible with our production environment